src/layouts/hass-error-screen.ts

```diff
@@ -9,24 +9,38 @@ import {
   TemplateResult,
 } from "lit-element";
 import { HomeAssistant } from "../types";
-import "./hass-subpage";
+import "../components/ha-menu-button";
+import "../components/ha-icon-button-arrow-prev";
 
 @customElement("hass-error-screen")
 class HassErrorScreen extends LitElement {
   @property({ attribute: false }) public hass!: HomeAssistant;
 
   @property({ type: Boolean }) public toolbar = true;
 
+  @property({ type: Boolean }) public rootnav = false;
+
+  @property() public narrow?: boolean;
+
   @property() public error?: string;
 
   protected render(): TemplateResult {
     return html`
       ${this.toolbar
         ? html`<div class="toolbar">
-            <ha-icon-button-arrow-prev
-              .hass=${this.hass}
-              @click=${this._handleBack}
-            ></ha-icon-button-arrow-prev>
+            ${this.rootnav
```

Comment:
Should we hide the "go back" button if `rootnav` ?

Reply:
Don't think so, you probably still came from somewhere?